extension/src/Utils/Logging/logListener.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+export interface ILogListener {

```

Comment:
@Eskibear is working on a common logger interface. Please hold this and use the new interface instead.